activesupport/lib/active_support/core_ext/string/inflections.rb

```diff
@@ -164,7 +164,7 @@ def deconstantize
   #
   #   <%= link_to(@person.name, person_path) %>
   #   # => <a href="/person/1-donald-e-knuth">Donald E. Knuth</a>
-  def parameterize(sep = '-')
+  def parameterize(sep = '-'.freeze)
```

Comment:
Let me clarify my first comment again:

I meant that it might be nice to have these tokens like `-`, `::`, `''` etc to, say, `DASH`,  `DOUBLE_COLON`, `EMPTY_STRING` etc defined in a separate file that's loaded upfront and those constants used everywhere instead of multiple `"-".freeze` calls all over the code.